modules/ROOT/pages/creating-an-odatav4-api-with-apikit.adoc

```diff
@@ -265,34 +367,25 @@ Returns five records starting from the second position.
 
 === Server-Side Pagination in APIkit for OData 4
 
-Your API implementation retrieves a limited number of records using a fix `pageSize` parameter. The response of the first request contains `@odata.nextlink` with a URL to get the next page.
+Server-side pagination is a way of controlling a subset of data requests that were fetched from a client-side server using a fixed `pageSize` parameter. It divides the response data set into discrete pages to improve data readability. The server defines the page size, which is the maximum number of records returned by the server in a request.
 
-For example:
+===== How server-side-pagination works in OData 4
```

Comment:
===== How Server-Side-Pagination Works in Odata 4 
(unless server-side-pagination is an XML element or code, in which case it stays lowercase)

Reply:
Title changed! Thanks!